src/game/Credits.cc

```diff
@@ -195,7 +195,7 @@ ScreenID CreditScreenHandle(void)
 
 static void InitCreditEyeBlinking(void);
 static void RenderCreditScreen(void);
-static void SelectCreditFaceMovementRegionCallBack(MOUSE_REGION* pRegion, INT32 iReason);
+static void SelectCreditFaceMovementRegionCallBack(MOUSE_REGION* pRegion, UINT32 iReason);
```

Comment:
Should the argument be renamed `uiReason` too?

Reply:
Yes, but I dont know how to easily achieve this without spending a lot of time manually renaming all of the occurences, so I left it.